### PR TITLE
Zeta/versioned transactions

### PR DESCRIPTION
**Pull request by kelvinlau20100**

### CODE REVIEW

File: ts/packages/anchor/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@coral-xyz/anchor",
```
Comment (filipzeta) [inline ts/packages/anchor/package.json]: Should we do zeta naming now? Probs just ctrl+f "coral" in the whole repo to double check

**Review by filipzeta — APPROVED** [review]

File: ts/packages/anchor/src/provider.ts [hunk 2]
```diff
@@ -17,6 +18,12 @@ import {
   SuccessfulTxSimulationResponse,
 } from "./utils/rpc.js";
 
+export function isVersionedTransaction(
+  transaction: Transaction | VersionedTransaction
+): transaction is VersionedTransaction {
+  return "version" in transaction;
```
Comment (xjcaa) [inline ts/packages/anchor/src/provider.ts:24]: what is in transaction, is there another way? im guessing this is like a dict?

**Comment by xjcaa** [comment]
side q: now that we have our own repo, can we port stuff like processTx into here from our sdk? we can make this one public

File: ts/packages/anchor/src/provider.ts (hunk 2 above)
Comment (kelvinlau20100) [reply ts/packages/anchor/src/provider.ts:24]: this is taken from  the solana web3 js library

**Comment by kelvinlau20100** [comment]
> side q: now that we have our own repo, can we port stuff like processTx into here from our sdk? we can make this one public

Yeah we can, I'll have a look at utils to see what we can port over

**Pull request merged**